style.css

```diff
@@ -531,3 +562,262 @@ footer .submit:active {
   background: #2230d2;
   color: #fff;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 1440px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .main-page {
+    height: 916px;
+    width: 1440px;
+    background-image: url("images/Header-bg.svg");
+    background-repeat: no-repeat;
+    background-position: 0;
+    padding-bottom: 114px;
+    border-bottom-right-radius: 30px;
+  }
+
+  .header {
+    width: 1440px;
+    height: 72px;
+  }
+
+  .header .toolbar .img-3 {
+    display: none;
+  }
+
+  .main-page .header .toolbar ul {
+    display: flex;
+    list-style-type: none;
+    margin-right: 154px;
+  }
+
+  .main-page .header .toolbar li {
+    font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] You don't need to write the same styling twice for the mobile and desktop versions. All the styling in the mobile version is applicable to the desktop version as well. You just need to write the layout **CHANGE** stylings for the desktop version inside the media query block of code. So please try to avoid writing the same code for the same element twice, inside and outside of the media query block.